### ISSUE TITLE
Enable copy and paste in the new registration form

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#3763]
A stub ticket for enabling a version of the copy and paste functionality present in the old registration form.

To be determined:

- handling errors, validation problems
- whether to report out to the form results page or the bulk action results page

**Comment by andrewjbtw** [comment]
Screenshot of @astridu 's design:

<img width="1117" alt="image (3)" src="https://user-images.githubusercontent.com/8975583/176951935-452df732-c05f-40fc-9ee5-2b68742dc59e.png">

For the help text, it looks like copy-and-paste from spreadsheet programs makes a tab-delimited list instead of comma-delimited, so that text should say "tab-delimited".

**Comment by andrewjbtw** [comment]
Also, we should use the order that was on the old grid because that's how spreadsheets are being lined up. But that's trickier than I thought because the druid column was in the middle.

I think if we do:

`| Barcode | Catkey | Source ID | Label`

It would approximate the old form. But users are going to have two tabs in a row between Source ID and Label if they do what they did before.

**Comment by andrewjbtw** [comment]
I made a screen recording of the copy-and-paste functionality and am posting here to get a shareable link. Since the video has no sound, this is what is happening below:

1. I filled out the registration form
2. I went to a spreadsheet to copy data to be used for registration
3. I returned to the registration form
4. I pasted in the data (this is where the paste box is highlighted)
5. The paste action populated the fields for the three items to be registered

https://user-images.githubusercontent.com/8975583/178391472-0bec6d24-1974-4345-b1fd-4379e9f4a35a.mov
